server/live_dash.py

```
@@ -0,0 +1,1278 @@
+        pid = o["pair_id"] or f"unpaired_{o['id']}"
+        if pid not in pairs_map:
+            pairs_map[pid] = {
+                "pair_id": pid,
+                "condition_id": o["condition_id"],
+                "max_pair_cost_at_post": o["max_pair_cost_at_post"],
+                "orders": [],
+            }
+        pairs_map[pid]["orders"].append(o)
+
+    pairs_list = []
+    for pid, pdata in pairs_map.items():
```

Ruff (0.16.1)

[warning] Loop control variable `pid` not used within loop body

(B007)